composer.json

```
@@ -20,7 +20,7 @@
     "guzzlehttp/promises": "0.1.1|^1.3",
     "squizlabs/php_codesniffer": "^3.5",
     "phpunit/phpunit": "^7.5||^8.5",
-    "phpspec/prophecy-phpunit": "^1.1",
+    "phpspec/prophecy-phpunit": "^2.0",
```

Let's see if this works
```suggestion
    "phpspec/prophecy-phpunit": "^1.1||^2.0",
```